### PR TITLE
Initial thoughts on features Transpower might like in OpenSCD

### PR DESCRIPTION
**Pull request by danyill**
Some initial thoughts.

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -25,3 +25,62 @@ Feeature ideas for the OpenSCD project. They are currently not prioritized.
...
+   - Having a plugin to automate deadbands by voltage level for SIPROTEC 5 devices
+   - Having a plugin to export relevant information to DCIM/IPAM tools (mainly IP addresses) so configuration can be built by importing it
+   - Having a plugin to export relevant information to Ansible network configuration scripting (to provide the information on what each device needs for VLANs, MAC address filtering, an augmented version of the Communication section)
+   - Automation of configuration for GOOSE and SV traffic based on known configurations of merging units, protection relays. We anticipate doing this by transforming the SLD into a graph where the nodes represent terminals and the edges represent `ConductingEquipment` etc and then carrying out a terminating depth-first search to identify e.g. CTs and CBs on a bus and provide subscriptions between e.g. bus protection, transformer protection and merging units. To achieve this we need:
+     - to be able to instantiate project icd files against LNodes within the substation section.
```
Comment (jarradraumati) [inline ROADMAP.md]:
```suggestion
     - to be able to bind instantiated IEDs `LN`s against `LNode`s within the substation/process section.
```

File: ROADMAP.md [hunk 2]
```diff
@@ -25,3 +25,62 @@ Feeature ideas for the OpenSCD project. They are currently not prioritized.
...
+     - Drag and drop of SLD symbols to create elements (from voltage level, bay, transformer, conducting equipment etc.) (including the `sxy` namespace coordinates)
+     - Ability to edit key parameters (voltage levels, descriptions)
+     - Ability to rotate symbols
+     - Ability to group objects, copy and paste and carry out basic alignment tasks (or to be connectable on a grid)
+     - Ability to "flip" a group of objects (to allow a bay to drawn in a particular direction -- Transpower draws bays to match the physical orientation of the site)
+     - Ability to verify/trace connectivity
+
+1. Substation Editor. The Substation editor provides much of the desired capability but does not yet allow:
+
+   - Connection of a an IED to a set of LNodes. (see [#1128](https://github.com/openscd/open-scd/issues/1128))
```
Comment (jarradraumati) [inline ROADMAP.md:67]:
```suggestion
   - Connection of an IED LNs to LNodes with the substation/process section. (see [#1128](https://github.com/openscd/open-scd/issues/1128))
```

File: ROADMAP.md [hunk 3]
```diff
@@ -25,3 +25,62 @@ Feeature ideas for the OpenSCD project. They are currently not prioritized.
...
+
+   - Being able to verify changes quickly and efficiently and handle collisions/merge differences if multiple people are working on the same project (currently we would plan to use git and plain-text differencing for management of SCL files)
+
+   - Being able to exchange one IED with another with a slightly different data model
+
+   Some initial prototyping and ideating done by Daniel/Christian (see also: [#896](https://github.com/openscd/open-scd/issues/896), [#669](https://github.com/openscd/open-scd/issues/669), [#892](https://github.com/openscd/open-scd/issues/892), [#349](https://github.com/openscd/open-scd/issues/349))
+
+1. Complete the publisher plugin 
+
+   - Provide the ability to add FCDOs to RCBs to allow MMS SCADA map augmentation ([#1092](https://github.com/openscd/open-scd/issues/1092))
```
Comment (jarradraumati) [inline ROADMAP.md:86]:
```suggestion
   - Provide the ability to add FCDAs to RCBs to allow MMS SCADA map augmentation ([#1092](https://github.com/openscd/open-scd/issues/1092))
```

**Review by jarradraumati — COMMENTED** [review]
Suggested some amendments.

File: ROADMAP.md (hunk 3 above)
Comment (danyill) [reply ROADMAP.md:86]: I think it's actually FCDOs. We can already do FCDAs in OpenSCD, but we can't drop entire data objects in like we do in other tools.

Comment (jarradraumati) [reply ROADMAP.md:86]: Ah alright. FCDO isn't mentioned within IEC 61850, so some may get confused. But it looks like other third parties have started using that term to define a functionally constrained data object.

This is the definition of FCDA:

> A reference to a single data attribute, a sub data object or sub data attribute of a data object having a specific functional constraint (FC) value shall be called functional constrained data attribute (FCDA).

**Comment by danyill** [comment]
This has served it's purpose. Closing.

**Pull request closed without merging**